### ISSUE TITLE
[python-package] Is it possible to get the current model's learning rate in a callback?

### ISSUE THREAD

**Issue by JivanRoquet** [dmlc/xgboost#1775]
When providing a function to the `callbacks` parameter of `xgb.train()` or `xgb.cv()`, is it possible that the model's current learning rate (or eta) be accessible to that function?

Currently, the callback function is provided with an `env` variable containing (in the case of `xgb.train()`) the model as a `xgb.Booster` object.

This object does not currently, as far as I know, provide any way of accessing the model's current learning_rate.

Two questions, then:

- is there a way to access `xgb.Booster`'s current learning rate that I haven't noticed?
- if not, does the C++ core library allow such parameter reading (so that it could be implemented fairly easily in Python's `xgb.Booster`?

**Comment by pommedeterresautee** [comment]
In last versions it s possible.
